File: Unit3/ViewBinding/app/src/main/java/com/example/viewbinding/MainActivity.java
package com.example.viewbinding;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.viewbinding.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(ActivityMainBinding.inflate(getLayoutInflater()).getRoot());
    }
}

File: Unit5/MyApplication/app/src/main/java/com/usu/myapplication/MainActivity.kt
package com.usu.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.usu.myapplication.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    var count = 0;
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.increment.setOnClickListener {
            count += 1
            binding.count.text = "" + count
        }

        binding.decrement.setOnClickListener {
            count -= 1;
            binding.count.text = "" + count
        }
    }
}